t/manifest.t

```diff
@@ -0,0 +1,15 @@
+#!perl -T
+use 5.006;
+use strict;
+use warnings;
+use Test::More;
+
+unless ( $ENV{RELEASE_TESTING} ) {
```

Comment:
This module is missing `.travis.yml`. It also should be registered with travis. In `.travis.yml` you should set this environment variable to make sure these tests are run.